### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MishaLehotskyi**
[DEMO LINK](https://MishaLehotskyi.github.io/layout_landing-page/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,289 @@
...
+                  name="message-textarea"
+                  class="contacts__textarea"
+                  placeholder="Message"
+                  required
+                ></textarea>
+
+                <button class="button contacts__button">Send</button>
+              </form>
+
+              <div class="contacts__data">
```
Comment (denys-danyliuk) [inline src/index.html:259]: 4'th point in checklist
Знімок екрана 2023-08-08 о 10 48 01

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,289 @@
...
+                </div>
+                <div class="contacts__data-piece">
+                  <h5 class="contacts__title">Address</h5>
+
+                  <a
+                    class="contacts__placeholder"
+                     href="https://goo.gl/maps/YqWaSi6UrgrqwFc3A"
+                  >
+                    400 first ave.<br>suite 700<br>Minneapolis, MN 55401
+                  </a>
```
Comment (denys-danyliuk) [inline src/index.html:278]: 6'th point in checklist

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,289 @@
...
+            <a class="top-bar__logo-link" href="#">
+              <img
+                src="images/logo-white.svg"
+                alt="MyBike logo"
+                class="top-bar__logo"
+              >
+            </a>
+
+            <div class="top-bar__icons">
+              <a href="#" class="icon icon--phone"></a>
```
Comment (denys-danyliuk) [inline src/index.html:36]: 5'th point in checklist

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,289 @@
...
+              Move Free
+            </h2>
+
+            <div class="about-us__content">
+              <p class="about-us__description">Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.</p>
+            </div>
+          </section>
+
+          <section class="compare-bikes compare-bikes--margin-bottom" id="compare-bikes">
+            <h2 class="section__title compare-bikes__title--margin-bottom">Compare bikes</h2>
```
Comment (denys-danyliuk) [inline src/index.html:116]: looks bad, 7'th point in checklist, images should be links also
Знімок екрана 2023-08-08 о 10 50 05

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,289 @@
...
+                  <img
+                    class="detail__photo"
+                    src="./images/details/detail-2.png"
+                    alt="Auto Unlock"
+                  >
+                </a>
+
+                <h3 class="detail__title">Auto Unlock</h3>
+                <p class="detail__description">The app senses when you're nearby to unlock automatically. GPS tracking so you know where your bike is and can track it anytime.</p>
+              </article>
```
Comment (denys-danyliuk) [inline src/index.html:177]: 7'th point in checklist

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,289 @@
...
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a class="top-bar__logo-link" href="#">
+              <img
+                src="images/logo-white.svg"
+                alt="MyBike logo"
+                class="top-bar__logo"
+              >
```
Comment (denys-danyliuk) [inline src/index.html:32]: this logo is too big on phone resolution, check the design

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,289 @@
...
+              alt="MyBike logo"
+              class="top-bar__logo"
+            >
+          </a>
+
+          <a href="#" class="icon icon--close"></a>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
```
Comment (denys-danyliuk) [inline src/index.html:63]: 12'th point in checklist

File: src/index.html [hunk 8]
```diff
@@ -1,13 +1,289 @@
...
+                <a href="#" class="button button__detail">Explore</a>
+              </div>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+            <h2 class="section__title contacts__title--margin-bottom">Contact us</h2>
+
+            <div class="contacts__wrapper">
+              <form
```
Comment (denys-danyliuk) [inline src/index.html:230]: 15'th point in checklist

File: src/index.html [hunk 9]
```diff
@@ -1,13 +1,289 @@
...
+                  </a>
+                </div>
+              </div>
+            </div>
+          </section>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer"></footer>
```
Comment (denys-danyliuk) [inline src/index.html:287]: 16'th point in checklist

**Review by denys-danyliuk — CHANGES_REQUESTED** [review]

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**